run.sh

```diff
@@ -2,9 +2,39 @@
 
 export HOME=/kibana
 
+: ${ELASTICSEARCH_URL:=http://localhost:9200}
 : ${ELASTICSEARCH_CERTIFICATE_VERIFICATION:=true}
+: ${ELASTICSEARCH_SSL_CA_CERT_PATH:=}
+: ${ELASTICSEARCH_USERNAME:=}
+: ${ELASTICSEARCH_PASSWORD:=}
+: ${XPACK_SECURITY_ENABLE:=false}
+: ${XPACK_SECURITY_ENCRYPTION_KEY:=$(uuidgen)}
```

Comment:
Yes. It's an arbitrary string of 32 characters or more used to encrypt credentials in a cookie. If it changes the worse that can happen is that user will be asked to sign in again.